improver_tests/ensemble_copula_coupling/ensemble_copula_coupling/test_ConvertProbabilitiesToPercentiles.py

```diff
@@ -41,23 +40,24 @@
 from iris.cube import Cube
 from iris.tests import IrisTest
 
-from improver.ensemble_copula_coupling.ensemble_copula_coupling import \
-    GeneratePercentilesFromProbabilities as Plugin
+from improver.ensemble_copula_coupling.ensemble_copula_coupling import (
+    ConvertProbabilitiesToPercentiles as Plugin)
 from improver.metadata.probabilistic import find_threshold_coordinate
 from improver.utilities.warnings_handler import ManageWarnings
```

Comment:
More of these import statements changing format

Reply:
Reverted.